rasa/nlu/featurizers/featurzier.py

```diff
@@ -10,6 +10,21 @@
 )
 
 
+def sequence_to_sentence_embedding(
+    features: Union[List[List[float]], scipy.sparse.spmatrix], method: Text = "mean"
+) -> Optional[np.ndarray]:
+    if features is None:
+        return None
+
+    if isinstance(features, scipy.sparse.spmatrix):
+        features = features.toarray()
+
+    if method == "mean" or method == "avg":
```

Comment:
I don't think we need to support this ambiguity, let's keep it `mean` and raise exception if the string is not supported